app/helpers/application_helper.rb

```diff
@@ -101,6 +101,10 @@ def tag_colors(tag)
     end
   end
 
+  def any_selfserve_auth?
```

Comment:
**suggestion (non-blocking):** I find the name a bit confusing, how about the more verbose `selfserve_auth_available?`?

Reply:
✅